### ISSUE TITLE
Error: Function "createCampaign" requires 6 arguments, but 0 were provided. Expected function signature: contract.call("createCampaign", _owner: string, _title: string, _description: string, _target: BigNumberish, _deadline: BigNumberish, _image: string): Promise<BigNumber>

### ISSUE THREAD

**Issue by Ash1shh** [adrianhajdin/project_crowdfunding#34]
Error: Function "createCampaign" requires 6 arguments, but 0 were provided.
Expected function signature: contract.call("createCampaign", _owner: string, _title: string, _description: string, _target: BigNumberish, _deadline: BigNumberish, _image: string): Promise<BigNumber>

**Comment by Div-Denis** [comment]
yes , I have a same question , I can see any value ,and also i can print value!,but just have that question

**Comment by Div-Denis** [comment]
I find the answer, at publicCampaign ,
try {
  ... 
 const data - await createCampaign({
      args:[
     ...
]
})
}

**Comment by tanayvaswani** [comment]
> I find the answer, at publicCampaign , try { ... const data - await createCampaign({ args:[ ... ] }) }

@Div-Denis can you please explain it in more detail?

**Comment by tanayvaswani** [comment]
@Div-Denis I got the answer, thanks!

**Comment by Div-Denis** [comment]
@tanayvaswani You can look at the contract you deployed in thirdweb, which is explained in detail
https://portal.thirdweb.com/react/react.usecontractwrite
@Ash1shh sorry man ,I used as much as I could, so I ran out of deployment

**Comment by arslanmaqsood713** [comment]
> 1. Go to your index.jsx file in the context folder.
> 2. Change the code in the file to this one:
>    const publishCampaign = async (form) => {
>    try {
>    const data = await createCampaign({args:[
>    address, //owner
>    form.title,
>    form.description,
>    form.target,
>    new Date(form.deadline).getTime(),
>    form.image
>    ]});
>    console.log("contract call success ", data);
>    } catch (error) {
>    console.log("contract call failed ", error);
>    }
>    }
> 3. The problem lies in the const data = await createCampaign();. It will be solved with the use of {args: [ ]}

Thanks it is working.

**Comment by Prasun60** [comment]
I have changed the index.js file in the context folder with the above mentioned method, but the the problem persist

**Comment by patooz** [comment]
@Div-Denis Works well thanks
